src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -24,9 +25,14 @@ public class BaseEppoClient {
           .registerModule(EppoModule.eppoModule()); // TODO: is this the best place for this?
 
   protected static final String DEFAULT_HOST = "https://fscdn.eppo.cloud";
-  protected final ConfigurationRequestor requestor;
 
-  private final ConfigurationStore configurationStore;
+  /**
+   * The Configuration Wrangler handles the complexities of loading configuration from multiple
+   * sources and saving to a persistent store. It contains the source of truth reference to the
+   * **Configuration** (UFC definitions and Bandit Parameters).
+   */
+  protected final ConfigurationWrangler configurationWrangler;
```

Comment:
not at all. I went with `Manager` because this class does more than just load configs (it has to manager, or wrangle the async config loading and apply an update strategy). tbh, I've just grown a little bored of `Manager` so look for ways to spice it up. maybe "Coordinator"...